data/mods/Magiclysm/field.json

```diff
@@ -11,5 +11,12 @@
     "half_life": "1 seconds",
     "phase": "solid",
     "display_field": false
+  },
+  {
+    "id": "fd_darkness",
+    "type": "field_type",
+    "intensity_levels": [ { "name": "darkness", "light_override": 0.0 } ],
+    "half_life": "10 seconds",
```

Comment:
just a heads up that the duration of the spell adds to the half-life, it doesn't override it.

Reply:
I'm not entirely sure how to change my values with that in mind, or what that would mean in practice and context. Do you have any suggestions off-hand?

Reply:
Oh, okay, I think I see what you're saying.  My intention was for the first level to give a minute of darkness, and the 30th to give fifteen. But if the `duration` adds to the `half_life`, that means that at first level, one minute after casting, the field is down to half intensity, then a quarter intensity after two minutes, and an eighth intensity after four.

So I should cut all my values in half.